collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -62,6 +65,9 @@ def _get_data(self):
         if self.alarm_contains_words != '':
             alarms = {alarm_name: alarms[alarm_name] for alarm_name in alarms for alarm_contains_word in
                       self.alarm_contains_words_list if alarm_contains_word in alarm_name}
+        if self.alarm_excludes_words != '':
+            alarms = {alarm_name: alarms[alarm_name] for alarm_name in alarms for alarm_excludes_word in
+                      self.alarm_excludes_words_list if not alarm_excludes_word in alarm_name}
```

Comment:
```suggestion
                      self.alarm_excludes_words_list if alarm_excludes_word not in alarm_name}
```

nitpick: this way looks slightly better for me.

Reply:
agree